persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -230,42 +268,50 @@ public static SyntaxTree updateConfigToml(Path configPath, List<String> names, S
                     moduleMembers = moduleMembers.add(member);
                 } else if (member instanceof TableNode) {
                     TableNode node = (TableNode) member;
-                    for (String schema : names) {
-                        if (schema.equals(packageName) && node.identifier().toSourceCode().trim().equals(
-                                packageName)) {
-                            existingNodes.add(schema);
-                            break;
-                        } else if (node.identifier().toSourceCode().trim().equals(packageName + "."
-                                + schema)) {
-                            existingNodes.add(schema);
-                            break;
+                    for (Path schemaPath : schemaPaths) {
+                        Path schemaFile = schemaPath.getFileName();
+                        if (Objects.nonNull(schemaFile)) {
+                            String schema = schemaFile.toString().replaceAll(".bal", "");
+                            if (schema.equals(packageName) && node.identifier().toSourceCode().trim().equals(
+                                    packageName)) {
+                                existingNodes.add(schema);
+                                break;
+                            } else if (node.identifier().toSourceCode().trim().equals(packageName + "."
+                                    + schema)) {
```

Comment:
Please check the logic? module name is not the schema filename now

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)